proto/v1beta1/cvss.proto

```diff
@@ -21,8 +21,7 @@ option java_multiple_files = true;
 option java_package = "io.grafeas.v1beta1.vulnerability";
 option objc_class_prefix = "GRA";
 
-// Common Vulnerability Scoring System version 3.
-// For details, see https://www.first.org/cvss/specification-document
+// Deprecated. See CVSS instead
```

Comment:
could we delete this altogether? v1beta1 implies some backwards-incompatibility is OK and it's better than have confusing similar-looking protos sprinkled through the definitions